public/js/strings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "debugger.breakpoints.header": "Breakpoints",
+  "debugger.callStack.header": "Call Stack",
+  "debugger.callStack.notPaused": "Not Paused",
+  "debugger.callStack.collapse": "Collapse Rows",
+  "debugger.callStack.expand": "Expand Rows",
+  "debugger.editor.searchResults": "%D of %D results",
+  "debugger.editor.noResults": "no results",
+  "debugger.scopes.header": "Scopes",
+  "debugger.scopes.notAvailable": "Scopes Unavailable",
+  "debugger.scopes.notPaused": "Not Paused",
+  "debugger.sources.header": "Sources",
+  "debugger.sources.search": "%S+P to search",
+  "debugger.watchExpressions.header": "Watch Expressions",
+  "debugger.commands.ignoreExceptions": "Ignore exceptions. Click to pause on uncaught exceptions",
+  "debugger.commands.pauseOnUncaughtExceptions": "Pause on uncaught exceptions. Click to pause on all exceptions",
+  "debugger.commands.pauseOnAllExceptions": "Pause on all exceptions. Click to ignore exceptions",
+  "debugger.commands.stepOut": "Step Out (%S%SF11)",
+  "debugger.commands.stepIn": "Step In (%SF11)",
+  "debugger.commands.stepOver": "Step Over (F10)",
+  "debugger.commands.resume": "Click to resume (F8)",
+  "debugger.commands.waitOnBreak": "Waiting for next execution",
+  "debugger.commands.pause": "Click to pause (F8)",
+  "debugger.welcome.search": "%S+P to search for files",
+  "debugger.sourceSearch.search": "Search...",
+  "debugger.sourceSearch.noResults": "No files matching %S found"
```

Comment:
not surprised, but there are lots of dupes in the properties file which should be reconciled...